packages/strapi-plugin-content-manager/admin/src/components/InputFileWithErrors/index.js

```diff
@@ -0,0 +1,122 @@
+/**
+ *
+ * InputFileWithErrors
+ */
+
+import React, { memo, useEffect, useState } from 'react';
+import { isArray } from 'lodash';
+import PropTypes from 'prop-types';
+
+import { Description, ErrorMessage, Label } from '@buffetjs/styles';
+import { Error } from '@buffetjs/core';
+import InputFile from './InputFile';
+import Container from './Container';
+
+function InputFileWithErrors({
+  className,
+  error: inputError,
+  inputDescription,
+  label,
+  multiple,
+  name,
+  onChange,
+  style,
+  validations,
+  value,
+}) {
+  const [fileLabel, setFilelabel] = useState(null);
+
+  useEffect(() => {
+    if (isArray(value) && value.length > 0) {
+      setFilelabel(1);
```

Comment:
FYI: Here you don't have an issue with the eslint rule because it knows that you are using a function inside your effect.

Reply:
Thanks